crates/typst-library/src/visualize/image/mod.rs

```diff
@@ -45,11 +45,11 @@ use crate::text::LocalName;
 /// ```
 #[elem(scope, Show, LocalName, Figurable)]
 pub struct ImageElem {
-    /// A [path]($syntax/#paths) to an image file or raw bytes making up an
-    /// image in one of the supported [formats]($image.format).
+    /// 画像ファイルへの[path]($syntax/#paths)、
+    /// あるいはサポートされている[format]($image.format)のいずれかで構成された画像を表す生のバイト列。
```

Comment:
たとえば以下のような訳ではいかがでしょうか？

```suggestion
    /// 画像ファイルへの[path]($syntax/#paths)、
    /// またはサポートされている[format]($image.format)の画像データの生バイト。
```